### ISSUE TITLE
Single Pass Instanced rendering shadowmap issue

### ISSUE THREAD

**Issue by Clavus** [ValveSoftware/unity-xr-plugin#4]
I get a strange shadowmap rendering artifact in-editor when using the OpenVR plugin with Single Pass Instanced as the Stereo Rendering Mode:
![image](https://user-images.githubusercontent.com/670995/86598622-a8bdc080-bf9d-11ea-91ed-65e82f526209.png)
Both eyes draw a large black plane over the surfaces at different offsets, the shadowmap itself appearss to be offset in the left eye as well.

For reference this is the (correct) image when I set it to Multi Pass:
![image](https://user-images.githubusercontent.com/670995/86598646-b410ec00-bf9d-11ea-9cf0-5f356b0cf216.png)

Unity 2019.4.0f1 and OpenVR XR Plugin 1.0.0 preview 4. Not using any render pipelines, and default Quality Settings. SteamVR Beta 1.13.9, using an Oculus Quest with a Link cable.

**Comment by 1runeberg** [comment]
Hi @Clavus - is this a test project you can share for testing?

**Comment by Clavus** [comment]
[OpenVRXRTest.zip](https://github.com/ValveSoftware/unity-xr-plugin/files/4879378/OpenVRXRTest.zip)
It's really nothing more than a brand new Unity project (3D preset), importing the OpenVR XR Plugin Installer package, setting up a few scene objects, and enabling the plugin.

I did note this error is spammed in the log while in Single Pass Instanced mode:
![image](https://user-images.githubusercontent.com/670995/86602167-8e3a1600-bfa2-11ea-9a84-5aabdec091f3.png)

**Comment by 1runeberg** [comment]
Thanks for the update @Clavus - which headset are you currently using?

**Comment by Clavus** [comment]
Oculus Quest connected with link cable, as mentioned in my original post.

**Comment by 1runeberg** [comment]
Thanks will have a look tomorrow, mustve seen your post re: headset prior to the edit ;)

**Comment by 1runeberg** [comment]
Hi @Clavus - can you try with at least Unity 2019.4.1f1 (LTS) or ideally the latest LTS - 2019.4.2f1 and double check latest releases as well for:

SteamVR beta: 1.13.9
Quest: 224383093

**Comment by 1runeberg** [comment]
and sorry, for the log spam, make sure you have a tracked pose driver to the Camera and not to another object in its parent hierarchy.

**Comment by Clavus** [comment]
After some digging I found the source of the problem: my work computer had outdated AMD drivers. After installing the latest driver the Single Pass Instanced rendering seems to work as intended. Sorry for the inconvenience!

As for the log spam: moving the camera component onto the tracked pose driver did not resolve that issue. I also compared it with using the Oculus XR Plugin and that did not cause any log spam, so I'm assuming this is could still be related to the OpenVR XR plugin. I'll make a separate issue report for this so we can close this one.
